config/common/server.yaml

```diff
@@ -17,7 +18,11 @@ deploymentConfigs:
   - metadata:
       name: "{{.ApplicationName}}-kieserver-{{$index}}"
       labels:
-        app: "{{.ApplicationName}}"
+        app: "{{.ApplicationName}}-kieserver-{{$index}}"
```

Comment:
won't this split the kieservers up into separate groups in the UI?

Reply:
Agree, I think this is a mistake.

Reply:
Yes, as Babak said , I’m grouping them by using different labels but this obe is a mistake